runtime/include/tasks/massivethreads/tasks-massivethreads.h

```diff
@@ -45,6 +45,9 @@ typedef struct {
 // between C code and Chapel code in the runtime.
 typedef intptr_t chpl_taskID_t;
 #define chpl_nullTaskID 0
+#ifndef CHPL_TASK_ID_STRING_MAX_LEN
+#define CHPL_TASK_ID_STRING_MAX_LEN 20
```

Comment:
Making it to 21 since the format string used is %PRIiPTR which prints it like normal integer.